Python/To-do List/README.md

```diff
@@ -0,0 +1,40 @@
+## README.md
+
+**Description:**
+
+With the rapid development of the world, everyone is engaged in a lot of work or even stuck with
+
+the thought of where to start from. To resolve this customary issue here presents ‘ON IT!!’.It is a
+
+python based GUI that helps boost productivity and helps get our work done in an organized
+
+way. It includes 4 fields to note the tasks to be performed today, tomorrow, the current week, or
+
+----------------------------------------------------------------------------------------------------------------------
+**Tech-Stack:**
+
+1.Python
+2.Tkinter
+
+---------------------------------------------------------------------------------------------------------------------------
+**Video:**
+
+[Video Link](https://drive.google.com/file/d/19s8E6qCUjR9Du1qNaTrbOPS3vwytF1es/view?usp=sharing)

```

Comment:
Add demonstration video here I request you to learn from these video and do necessary 

https://drive.google.com/file/d/12cjBwDaTCK4pu7cqNI_FNuRAZwamCywP/view?usp=sharing